src/style/app.module.css

```diff
@@ -573,9 +627,9 @@
   --bs-btn-active-bg: var(--search-button-bg);
   --bs-btn-active-border-color: var(--search-button-bg);
   margin-bottom: 10px;
-  color: var(--search-button-fill);
-  background-color: var(--search-button-bg);
-  border-color: var(--search-button-bg);
+  background-color: var(--search-button-bg) !important;
+  border-color: var(--search-button-bg) !important;
+  color: var(--brown-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`